### PR TITLE
Use the solr:slim docker image

### PR DESCRIPTION
**Pull request by rotated8**
The previous image, `solr:7-alpine` has not been updated in almost a year. If we believe our components that use Solr are not tied to Solr 7, we should allow the Solr version to advance. With the solr_version parameter, the orb's consumers can control which version they wish to use.

The README has been updated to reflect this.

### CODE REVIEW

**Comment by rotated8** [comment]
Fixes #33

**Comment by rotated8** [comment]
From previous discussion in Slack, @no-reply said "+0 to dropping the solr version from the spec. i think we should strongly recommend users override the default to pin the image they use more tightly, if we loosen the orb default"

File: src/executors/ruby_fcrepo_solr.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ parameters:
     default: '8985'
   solr_version:
     type: string
-    default: '7-alpine'
+    default: 'slim'
```
Comment (jrgriffiniii) [inline src/executors/ruby_fcrepo_solr.yml:17]: ❤️

**Comment by cjcolvar** [comment]
This looks great @rotated8! Do you think it is ready to be merged now?

**Pull request merged**